### PR TITLE
[12.0] [ADD] fieldservice_operating_unit

### PR DESCRIPTION
**Pull request by nikul-serpentcs**
* This module adds operating unit information to Field Service orders.

Issue #214

### CODE REVIEW

File: fieldservice_operating_unit/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* Nikul Chaudhary <nikul.chaudhary.serpentcs@gmail.com>
+* Maxime Chambreuil <mchambreuil@opensourceintegrators.com>
+
```
Comment (max3903) [inline fieldservice_operating_unit/readme/CONTRIBUTORS.rst]:
```suggestion
```

File: fieldservice_operating_unit/readme/CREDITS.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+* Open Source Integrators
+* Serpent Consulting Services Pvt. Ltd.
+
```
Comment (max3903) [inline fieldservice_operating_unit/readme/CREDITS.rst]:
```suggestion
```

File: fieldservice_operating_unit/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,2 @@
+This module adds operating unit information to Field Service orders
+
```
Comment (max3903) [inline fieldservice_operating_unit/readme/DESCRIPTION.rst]:
```suggestion
```

File: fieldservice_operating_unit/readme/USAGE.rst [hunk 4]
```diff
@@ -0,0 +1,4 @@
+* Go to Field Service
+* You only see the FSM orders of your operating units
+* Create an order. It is assigned to your default operating unit.
+
```
Comment (max3903) [inline fieldservice_operating_unit/readme/USAGE.rst]:
```suggestion
```

**Review by max3903 — APPROVED** [review]
Minor changes. :+1:

**Review by murtuzasaleh — APPROVED** [review]

**Pull request merged**